psa-ext/cert-meta.cddl

```diff
@@ -0,0 +1,16 @@
+$$endorsed-value-map-extension //= (
+  2 => psa-cert-meta
+)
+
+psa-cert-meta = {
+    1  => text,   ; cert-level PSA Certification level
+    2  => text,   ; cert-num A unique number for each certificate
```

Comment:
sounds like we need to be more precise here in terms of what we expect (RFC3339 sounds like the sensible choice)